code-scanning/codacy.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       # Execute Codacy Analysis CLI and generate a SARIF output with the security issues identified during the analysis
       - name: Run Codacy Analysis CLI
-        uses: codacy/codacy-analysis-cli-action@d840f886c4bd4edc059706d09c6a1586111c540b
+        uses: codacy/codacy-analysis-cli-action@v4
```

Comment:
Can you put the sha instead of the version number? Thanks

Reply:
ok

Reply:
@lolgab Pinging again. Kindly update this.